engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -8,6 +8,11 @@ <h2 translate>Content Host Registration</h2>
 
   <div data-block="content">
     <p translate>To register a content host to this server, follow these steps.</p>
+    <p bst-alert="warning" >
+      <span translate>
+        In a future version of Katello, we are planning to deprecate the <b>katello-ca-consumer</b> package and <b>bootstrap.py</b>. Use the new registration method <a href="https://{{ katelloHostname }}/hosts/register">Register Host</a>
```

Comment:
I'd tweak the wording slightly to indicate that at this time the old methods are now (technically) deprecated (correct me if wrong!).

`
The following instructions are obsolete because registering with the *katello-ca-consumer* package and *bootstrap.py* have been deprecated and will be removed in a future version. They have been replaced by the *Register Host* workflow. Registering new hosts with the below instructions is not recommended.
`

I also removed the 'Katello' usage to make this more appropriate for the downstream. Whatcha think?

Reply:
> we are planning to deprecate the <b>katello-ca-consumer</b> package

The [global registration template](https://github.com/theforeman/foreman/blob/fce22f37651ba76a42c2ef6385342af4fb84bb12/app/views/unattended/provisioning_templates/registration/global_registration.erb#L74) still downloads the consumer rpm, no? 
```bash
CONSUMER_RPM=$(mktemp --suffix .rpm)
    curl --output $CONSUMER_RPM <%= subscription_manager_configuration_url(hostname: @url_host) %>
    yum localinstall $CONSUMER_RPM -y
```

Reply:
Bootstrap should only handle the cases GR can't. So soft deprecated? ;)

Reply:
@jeremylenz that makes the wording harder now lol. What do you think about adding something like:

```bash
Register Host is the new preferred way of registering a Content Host. Use Register Host <link>
```

Reply:
I like the direction!  It feels better to me not to style it as a "warning," but just letting users know about a new way to do it.  So the style of the banner can be "info" instead of "warning". And not mentioning bootstrap or deprecation makes the messaging a lot smoother.

As for the wording, how about
```
NEW: To register a new content host without following these manual steps, see <a>Register Host</a>.
```

Reply:
Works for me, will get it updated.

Reply:
I like @jeremylenz suggestion!

Reply:
Updated! screenshot:

https://nimb.ws/9InBbZ